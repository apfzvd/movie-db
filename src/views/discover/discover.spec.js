import React from 'react'
import { shallow } from 'enzyme'

import Discover from './discover-component'

const props = {}

describe('Discover component', () => {
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(<Discover {...props} />)
      expect(wrapper.debug()).toMatchSnapshot()
    })
  })
})
