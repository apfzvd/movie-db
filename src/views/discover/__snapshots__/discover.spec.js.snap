// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Discover component #render render correctly 1`] = `
"<div className={[undefined]}>
  <h1 className={[undefined]}>
    Descobrir
  </h1>
  <section className={[undefined]}>
    <Shelf displayInfo=\\"clean\\" title=\\"Nos cinemas agora\\" request=\\"getNowPlaying\\" slidesToShow={2} showExtra={true} tileOrientation=\\"portrait\\" />
  </section>
  <section className={[undefined]}>
    <Shelf displayInfo=\\"clean\\" title=\\"Em breve nos cinemas\\" request=\\"getUpcoming\\" slidesToShow={2} showExtra={true} tileOrientation=\\"portrait\\" />
  </section>
</div>"
`;
